src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -479,11 +485,15 @@ private synchronized int execMatlabCommand(Run<?, ?> build, FilePath workspace,
         return postRunnerSwitch;
     }
 
-    private String[] getRunnerSwitch() {
+    private String[] getRunnerSwitch() throws IOException, InterruptedException {
         final String runCommand;
         final String testRunMode = this.getTestRunTypeList().getDescriptor().getDisplayName();
         if (!testRunMode.equalsIgnoreCase(
                 Message.getValue("builder.matlab.customcommandoption.display.name"))) {
+            
+            //Copy MATLAB scratch file into the workspace only if Automatic option is selected.
```

Comment:
A method called getRunnerSwitch (whose responsibility is to provide the switches used by runner) is doing a copy operation for setting up the artifacts required to run the tests. This seems odd.